prereise/gather/demanddata/transportation_electrification/tests/test_immediate_charging.py

```diff
@@ -0,0 +1,151 @@
+import inspect
+import os
+
+import numpy as np
+
+import prereise
+import prereise.gather.demanddata.transportation_electrification.immediate_charging_HDV as immediate_charging_HDV
+from prereise.gather.demanddata.transportation_electrification.data_helper import (
+    load_urbanized_scaling_factor,
+)
+from prereise.gather.demanddata.transportation_electrification.immediate import (
+    immediate_charging,
+)
+
+
+def test_immediate_charging_region1():
+    immediate_charging(
+        census_region=1,
+        model_year=2017,
+        veh_range=100,
+        kwhmi=0.242,
+        power=6.6,
+        location_strategy=2,
+        veh_type="LDV",
+        filepath=os.path.join(
+            os.path.dirname(inspect.getsourcefile(prereise)),
+            "gather",
+            "demanddata",
+            "transportation_electrification",
+            "data",
+            "nhts_census_updated_dwell",
+        ),
+    )
+
+
+def test_immediate_charging_mdv():
+
+    result = immediate_charging_HDV.immediate_charging(
+        model_year=2050,
+        veh_range=200,
+        power=80,
+        location_strategy=1,
+        veh_type="MDV",
+        filepath=os.path.join(
```

Comment:
Should we define the root data folder (e.g. `$root/prereise/gather/demanddata/transportation_electrification/data`) in a const.py, or maybe a module level variable? Since it's reused a few times.